Packs/IntSight/Integrations/Rapid7ThreatCommand/Rapid7ThreatCommand.py

```diff
@@ -0,0 +1,5325 @@
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    alert_id = args["alert_id"]
+    client.add_alert_note(
+        alert_id=alert_id,
+        note=args["note"],
+        file_entry_ids=argToList(args.get("entry_ids")),
+    )
+    return CommandResults(
+        readable_output=ReadableOutputs.ALERT_ADD_NOTE.value.format(alert_id)
+    )
+
+
+def get_alert_blocklist_status_command(
+    client: Client, args: dict[str, Any]
+) -> CommandResults:
+    """
+    Get alert's blocklist status.
+
+    Args:
+        client (Client): Threat Command API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    alert_id = args["alert_id"]
+
+    response = client.get_alert_blocklist(alert_id=alert_id)
+    mapped_response = [dict_to_lowercase(msg) for msg in response]
+    outputs = {"id": alert_id, "BlockList": mapped_response}
+    return command_result_generate(
+        readable_message=ReadableOutputs.ALERT_BLOCKLIST_GET.value.format(alert_id),
+        outputs=outputs,
+        headers=["value", "status"],
+        prefix="Alert",
+        key_field="id",
+        raw_response=response,
+        readable_outputs=mapped_response,
+    )
+
+
+def update_alert_blocklist_command(
+    client: Client, args: dict[str, Any]
+) -> CommandResults:
+    """
+    Change selected IOCs blocklist status.
+
+    Args:
+        client (Client): Threat Command API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    validate_update_alert_blocklist(args)
+
+    alert_id = args["alert_id"]
+    blocklist_status = args["blocklist_status"]
+
+    client.update_alert_blocklist(
+        alert_id=alert_id,
+        domains=argToList(args.get("domains")),
+        urls=argToList(args.get("urls")),
+        ips=argToList(args.get("ips")),
+        emails=argToList(args.get("emails")),
+        blocklist_status=blocklist_status,
+    )
+
+    return CommandResults(
+        readable_output=ReadableOutputs.ALERT_BLOCKLIST_UPDATE.value.format(
+            blocklist_status
+        )
+    )
+
+
+def list_alert_image_command(
+    client: Client, args: dict[str, Any]
+) -> List[CommandResults | List[Dict[str, Any]]] | CommandResults:
+    """
+    List alert images by alert ID.
+
+    Args:
+        client (Client): Threat Command API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    images = []
+    alert_id = args["alert_id"]
+    complete_alert = client.get_alert(alert_id=alert_id)
+    img_ids = dict_safe_get(complete_alert, ["Details", "Images"], [])
+    images = [
+        fileResult(filename=f"{img}.png", data=client.get_alert_image(img).content)
```

Comment:
please add `file_type=EntryType.ENTRY_INFO_FILE` to fileResult (meaning this file is not an IOC)